ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayDeviceLocationWithNmeaDataSources/DisplayDeviceLocationWithNmeaDataSources.cpp

```diff
@@ -101,37 +108,40 @@ void DisplayDeviceLocationWithNmeaDataSources::start()
 
   m_mockDataIterator = 0;
   m_timer->start(1000);
-  m_mapView->locationDisplay()->setAutoPanMode(LocationDisplayAutoPanMode::Recenter);
 }
 
-void DisplayDeviceLocationWithNmeaDataSources::reset() {
+void DisplayDeviceLocationWithNmeaDataSources::reset()
+{
   // Disable simulated data
   m_timer->stop();
   disconnect(m_timer, &QTimer::timeout, nullptr, nullptr);
 
   // Stop displaying user's location
   m_mapView->locationDisplay()->stop();
+
   // Stop receiving location data
   m_nmeaLocationDataSource->stop();
-  m_mapView->locationDisplay()->setAutoPanMode(LocationDisplayAutoPanMode::Off);
 }
 
-bool DisplayDeviceLocationWithNmeaDataSources::loadMockDataFile(QString filePath)
+bool DisplayDeviceLocationWithNmeaDataSources::loadMockDataFile(const QString &filePath)
```

Comment:
this is the style we try to use:

```suggestion
bool DisplayDeviceLocationWithNmeaDataSources::loadMockDataFile(const QString& filePath)
```